### DISCUSSION TITLE
kiwi.extract_words(inputs) 실행시,에러

### DISCUSSION THREAD

**Discussion by sfjun** [bab2min/kiwipiepy#59] [Q&A]
오늘 막 kiwipiepy를 시작했습니다. 

kiwipiepy 버전 0.7.2, linux 
설치후 "안녕" 테스트까지 문제가 없습니다. 출력은 좀 다르지만요

첫번째 .extract_words(inputs)을 실행하면 
TypeError "extractWords requires 1st parameter which is callable"
나와서 inputs을 다양하게 변경해서 해 봤는데 같은 에러 입니다.
(단순 리스트, iter(inputs)... 파일로 만들어서...)
-->inputs,이나 text.txt 예시를 알 수 있을까요. 

두번째는 kiwi.extract_words(IterableTextFile('xx.txt')...) 실행하면
NameError: name 'IterableTextFile' is not defined
-->설치나 버전에 문제가 있는건가요?

같은 고민으로 해결하신분 있으시면 의견 부탁드립니다.

**Comment by bab2min** [comment]
안녕하세요, @sfjun
0.7.2버전말고 최신버전인 0.10.1 버전을 설치하시길 권장드립니다. 0.10.0버전부터 해당 함수의 사용방법이 변경되었고, 현재 README의 예제는 0.10.0버전 기준으로 작성되어있기 때문입니다.

0.7.2버전으로 진행하시겠다면 https://github.com/bab2min/kiwipiepy/tree/v0.7.2 페이지의 예제를 참조하시면 되겠습니다.
